### PR TITLE
Configure goodcheck.yml

### PR DESCRIPTION
**Pull request by TMcMeans**
For testing finding instances of .only hanging in testing suites!

MVP worthy task for enabling goodcheck/slider as a viable PR bot (w/ customizable regexp rules!) in our codebase if it works.

### CODE REVIEW

File: apps/manage/test/lib/assign.test.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as AssignModel from '../../lib/view-model/assign'
 import { filterAssignedItems } from '../../lib/view-model'
 
 describe('filterAssignedItems', () => {
-  test('filters the "items" list by the "assigned" list', () => {
+  test.only('filters the "items" list by the "assigned" list', () => {
```
Comment (TMcMeans) [inline apps/manage/test/lib/assign.test.js]: > __[Goodcheck]__ Please remove any instances of .only from tests in the test file you are working in. ([link](https://sider.review/gh/repos/136378049/pulls/1195#issue-4619643))

test

File: apps/onboarding/test/lib/view-model.test.js [hunk 2]
```diff
@@ -732,7 +732,7 @@ describe('isConsentValid', () => {
   })
 })
 
-test('setPersonAttr returns areClassInfoFieldsValid if isTransferClient is true', () => {
+test.only('setPersonAttr returns areClassInfoFieldsValid if isTransferClient is true', () => {
```
[the remark this answers is not in the document]
Comment (TMcMeans) [reply apps/onboarding/test/lib/view-model.test.js]: This bot is super dope! It automatically comments on your PRs after targeting any issues found in your code.

**Comment by TMcMeans** [comment]
Requesting review so ya'll can see the WIP but please don't merge yet! I would like to test another possible use case before I know for sure its worth adding to our codebase.

File: apps/onboarding/test/lib/view-model.test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (lionsdwarf) [reply apps/onboarding/test/lib/view-model.test.js]: Dopesauce! Well done 👏

File: goodcheck.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      - token: describe.only(
+      - token: it.only(
+    glob: 
+      - "**/*.test.js"
+    message: |
+      Please remove any instances of .only from tests in the test file you are working in.
+    pass:
+      - test(
+      - describe(
+      - it(
```
Comment (lionsdwarf) [inline goodcheck.yml]: Just curious...what's the point of specifying these `pass` values?

Comment (TMcMeans) [reply goodcheck.yml]: Those pass values are there to give goodcheck customized examples of how to pass the test.

Comment (lionsdwarf) [reply goodcheck.yml]: I guess I'm confused why we want it to scan for things that will pass.

Comment (TMcMeans) [reply goodcheck.yml]: It's so that we can see the little green checkmark for approval when merging PRs.

**Comment by craigspaeth** [comment]
Yay! So excited to merge this! 👏

**Pull request merged**